File: src/main/java/com/DavidM1A2/AfraidOfTheDark/common/block/BlockGnomishMetalPlate.java
/*
 * Author: David Slovikosky
 * Mod: Afraid of the Dark
 * Ideas and Textures: Michael Albertson
 */
package com.DavidM1A2.AfraidOfTheDark.common.block;

import com.DavidM1A2.AfraidOfTheDark.common.block.core.AOTDBlock;

import net.minecraft.block.material.Material;

public class BlockGnomishMetalPlate extends AOTDBlock
{
	public BlockGnomishMetalPlate()
	{
		super(Material.rock);
		this.setUnlocalizedName("gnomishMetalPlate");
		this.setHardness(2.0F);
		this.setResistance(10.0F);
		this.setHarvestLevel("pickaxe", 2);
	}
}

File: src/main/java/com/davidm1a2/afraidofthedark/common/block/BlockGnomishMetalPlate.kt
package com.davidm1a2.afraidofthedark.common.block

import com.davidm1a2.afraidofthedark.common.block.core.AOTDBlock
import net.minecraft.block.material.Material

/**
 * Class representing the gnomish metal plate
 *
 * @constructor sets the name and block properties
 */
class BlockGnomishMetalPlate : AOTDBlock("gnomish_metal_plate", Material.ROCK)
{
    init
    {
        setHardness(2.0f)
        setResistance(10.0f)
        this.setHarvestLevel("pickaxe", 2)
    }
}